### ISSUE TITLE
Resizable main window

### ISSUE THREAD

**Issue by basxto** [MikeMaximus/gbm#77]
The main window is not resizable, which is a bit inconvinient for reading the logs.
It would be nice to change that, also would work better with [tiling window managers](https://en.wikipedia.org/wiki/Tiling_window_manager), who start applications maximized.

Distribution: Archlinux
Kernel: 4.11.3
mono: 5.0.0.100
mono-basic: 4.6
p7zip: 16.02
sqlite: 3.19.3
bash: 4.4.012

**Comment by MikeMaximus** [comment]
I'll play with the resizing options for the next version and see how it goes.

**Comment by MikeMaximus** [comment]
This is going to require a re-design of that form to work around weird issues Mono has with resizing/anchoring/docking controls.  

May not make it into 1.02.

**Comment by MikeMaximus** [comment]
I got this figured out now.  Mono has issues if you programmatically modify the form size at run-time and anchor controls, causing it to extend controls beyond the form size and even crash.

So I've done the following:

1.  Allowed resizing and maximizing the main form.
2.  Anchored the log so it will resize with the form.
3.  Removed the "Show/Hide Log" button and the need to programmatically modify the form size. 
4.  Enforced a minimize size which allows you to nicely hide the log with a quick resize.

So far it's working well on both Windows and Linux.
